app/assets/stylesheets/base/_forms.scss

```diff
@@ -1,6 +1,6 @@
 input {
   width: 100%;
-  padding: rem(5px);
+  padding: 5px;
```

Comment:
I think the reason for removing `rem` out because it will return `0.3125`, right?
How about using some evens number such as `6` or `8` instead? 😅

Reply:
True I'll have it changed 😄 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/35/commits/4de259ac3b8d20a892a061fd18c97a8a63517e55